deploy/docker-compose-example.yml

```
@@ -43,3 +43,6 @@ volumes:
   edumfa-data:
```

```suggestion
  edumfa-config:
```
Also changing it to `-config` here